ts/utils/bottomSheet.ts

```diff
@@ -0,0 +1,39 @@
+import { BottomSheetModalConfigs } from "@gorhom/bottom-sheet/lib/typescript/types";
+import * as React from "react";
+import { AccessibilityContent } from "../components/bottomSheet/AccessibilityContent";
+import { BlurredBackgroundComponent } from "../components/bottomSheet/BlurredBackgroundComponent";
+import { BottomSheetContent } from "../components/bottomSheet/BottomSheetContent";
+import { BottomSheetHeader } from "../components/bottomSheet/BottomSheetHeader";
+import { isScreenReaderEnabled } from "./accessibility";
+
+export type BottomSheetProps = {
+  content: React.ReactNode;
+  config: BottomSheetModalConfigs;
+};
+
+export const bottomSheetContent = async (
+  content: React.ReactNode,
+  title: string,
+  snapPoint: number,
+  onClose: () => void
+): Promise<BottomSheetProps> => {
+  const isScreenReaderActive = await isScreenReaderEnabled();
+
+  const header = BottomSheetHeader({ title, onClose });
+
+  const bottomSheetBody: React.ReactNode = isScreenReaderActive
+    ? AccessibilityContent(header, content)
+    : BottomSheetContent({ children: content });
+
+  return {
+    content: bottomSheetBody,
+    config: {
+      snapPoints: [snapPoint],
```

Comment:
Since there are multiple screen resolutions could be an improvement check if the given snap point is greater than the screen height?

```typescript
const screenHeight = Dimensions.get("window").height;
// make sure the given point don't exceed the screen height
const snapPoints = [snapPoint].map(sp => Math.min(sp, screenHeight));
```

Reply:
@Undermaken It could be an improvement of course, maybe we should consider using the string approach with percentages which is provided by the library https://github.com/gorhom/react-native-bottom-sheet#snappoints

What do you think?

cc. @fabriziofff